integration_tests/tests/test_dbt_artifacts/test_microbatch_compiled_code.py

```
@@ -0,0 +1,99 @@
+from contextlib import contextmanager
+from pathlib import Path
```

Flake8 (7.3.0)

[error] 'pathlib.Path' imported but unused

(F401)